examples/28-wireframe/vs_wf_wireframe_bc.sc

```diff
@@ -0,0 +1,18 @@
+$input a_position
+$output v_view
+
+/*
+ * Copyright 2016 Dario Manesku. All rights reserved.
+ * License: https://github.com/bkaradzic/bgfx/blob/master/LICENSE
+ */
+
+#include "../common/common.sh"
+#include "uniforms.sh"
+
+void main()
```

Comment:
If there is no difference in shaders, you should use existing ones instead to avoid duplication.

Reply:
There's a difference here where `bc` is defined and used instead on the `v_bc`:
```
    vec3 bc = gl_BaryCoord;
    vec3 fw = abs(dFdx(bc)) + abs(dFdy(bc));
    vec3 val = smoothstep(vec3_splat(0.0), fw*thickness, bc);
```